Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: deploy-image deploy run-migrations
+
+SHORT_SHA=$(shell git rev-parse --short=7 HEAD)
+APP_IMAGE_NAME=gcr.io/oursky-kube/gitlint-bot:${SHORT_SHA}
```

Comment:
Put the image repo in a separate variable. We should tag the image with both:
- `latest`
- the short commit hash